pkg/operator/operator.go

```diff
@@ -452,6 +464,75 @@ func (c *csiDriverOperator) isCSIDriverInUse() (bool, error) {
 	return false, nil
 }
 
+func (c *csiDriverOperator) checkPrereq(instance *v1alpha1.AWSEBSDriver) error {
+	return c.checkPreviousInstall(instance)
+}
+
+func (c *csiDriverOperator) checkPreviousInstall(instance *v1alpha1.AWSEBSDriver) error {
+	// Check that there is no other CSI driver installation.
+	// Get the driver name
+	expectedDriver := resourceread.ReadCSIDriverV1Beta1OrDie(generated.MustAsset(csiDriver))
+	driverName := expectedDriver.Name
+	if err := c.checkPreviousCSIDriver(driverName); err != nil {
+		return err
+	}
+
+	// CSIDriver does not exist. Check if the operand namespace exists.
+	_, err := c.namespaceInformer.Lister().Get(operandNamespace)
+	if err == nil {
+		klog.V(4).Infof("Namespace %s exists", operandNamespace)
+
+		// OCP operand namespace exists. If there is a CSI driver installed, it must be the OCP one.
+		return nil
+	}
+	if !errors.IsNotFound(err) {
+		return fmt.Errorf("error getting Namespace %s: %s", operandNamespace, err)
+	}
+	klog.V(4).Infof("Namespace %s does not exist", operandNamespace)
+
+	// OCP operand namespace does not exist. Check if the driver is installed by inspecting all CSINodes
+	// (that's why we saved this loop to the last possible moment).
+	csiNodes, err := c.csiNodeInformer.Lister().List(labels.Everything())
```

Comment:
new rbac permissions?

Reply:
Yes, CSINode is new... How did the test pass?

Reply:
The operator can list / watch CSINodes to be able to grant these permissions to external-attacher role. So no new RBAC rules are necessary.

BTW, good catch.